packages/scene-composer/src/store/slices/SceneDocumentSlice.spec.ts

```diff
@@ -561,7 +561,8 @@ describe('createSceneDocumentSlice', () => {
         };
         const draft = { lastOperation: undefined, document };
         const getSceneNodeByRef = jest.fn().mockReturnValue(document.nodeMap.testNode);
-        const get = jest.fn().mockReturnValue({ getSceneNodeByRef }); // fake out get call
+        const getSceneProperty = jest.fn().mockReturnValue();
+        const get = jest.fn().mockReturnValue({ getSceneNodeByRef, getSceneProperty }); // fake out get call
```

Comment:
should add a test case for checking `updateEntity` is called when has layerId

Reply:
done